scripts/aws/setupdb.sh

```diff
@@ -69,7 +69,10 @@ envdir /etc/mmw.d/env /opt/app/manage.py migrate
 
 function download_and_load {
     for f in "${FILES[@]}"; do
-        curl -s $FILE_HOST/$f | gunzip -q | psql --single-transaction
+        rm -f tmp.sql tmp.sql.gz
```

Comment:
I think there's several ways we can enhance this to make it more robust but I'd like to do that in a separate PR. The only reason I included this change is in case somebody needs to populate their workstation with boundary layers and runs into the same problems that I encountered. It probably makes more sense to just revert this since I don't know what impact it will have on deployments.